test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,573 @@
+package functional
```

Comment:
Will this link to a test plan? 
If it does we should leave some developer docs here pointing to the test plan, if not we should add developer docs here outlining what resources we expect to find in a standalone vpc. I think it would help with debugging issues here. What do you think?

Reply:
This will/should be linked to a test plan ID, see comment above